crates/matrix-sdk/src/sliding_sync/mod.rs

```diff
@@ -277,6 +277,8 @@ impl SlidingSync {
 
         let must_process_rooms_response = self.must_process_rooms_response().await;
 
+        debug!(yes = must_process_rooms_response, "Must process rooms response?");
```

Comment:
(uber nit) does it need debug or could it be a trace?

Reply:
Changed to `trace!`. I've rebased the commits. Gonna merge.